content/platforms/guides/using-kustomize.md

```diff
@@ -0,0 +1,137 @@
+For example, for development uses, a customer may want to set the resource requests and limits for pods to smaller values or scale back the number of nodes. At the same time, they may want a shared set of names, features, and secret usage for all deployments. This is just the kind of situation that kustomize is designed to handle well.
+
+## A simple example
+
+Kustomize works on the building variants from a hierarchy of configurations. In the simplest setup, there is one “base” directory containing all the common resources. For each variant, there are different directories each with instructions on how to produce the variant.
+
+For example, say we simply want to control the the limits and requests for a node so that they are different values for development and production. To start, we must create a “base” directory that contains a default configuration for a simple cluster:
+
+```
+mkdir -p base
+cat << EOF > base/cluster.yaml
+apiVersion: app.redislabs.com/v1
+kind: RedisEnterpriseCluster
+metadata:
+  name: test
+spec:
+  nodes: 3
+  license: "xyzzy"
+  username: "app@example.org"
+EOF
+```
+
+We then need to configure the “base” directory to be recognized by kustomize. The simplest configuration just lists the resources:
+
+```
+cat << EOF > base/kustomization.yaml
+resources:
+ - cluster.yaml
+EOF
+```
+
+Now we can create a variant for development purposes that sets the resource limits, requests, and changes the namespace:
+
+```
+mkdir -p dev
+cat << EOF > dev/cluster.yaml
+apiVersion: app.redislabs.com/v1
+kind: RedisEnterpriseCluster
+metadata:
+  name: test
+spec:
+  redisEnterpriseNodeResources:
+    limits:
+      cpu: 1
+      memory: 3Gi
+    requests:
+      cpu: 1
+      memory: 3Gi
+EOF
+```
+
+When we created the resource, we need to repeat the prolog metadata so that kustomize can match our resources. At minimum, we need the apiVersion, kind, and metadata.name.
+
+As before, we need to tell kustomize about our resources via a kustomization.yaml file. In this case, we need to tell kustomize what to do with our variant of cluster.yaml. In this case, we want to patch the base cluster.yaml via a strategic merge of properties:
+
+```
+cat << EOF > dev/kustomization.yaml
+bases:
+ - ../base
+
+namespace: dev
+
+patchesStrategicMerge:
+ - cluster.yaml
+EOF
+```
+
+At this, we can test our “dev” customization via:
+
+```
+kubectl kustomize dev
+```
+
+and the output should look something like:
+
+```
+apiVersion: app.redislabs.com/v1
+kind: RedisEnterpriseCluster
+metadata:
+  name: test
+  namespace: dev
+spec:
+  license: xyzzy
```

Comment:
```suggestion
```
Remove because this is mal formatted and as is will generate an error.

Reply:
Removed.